monkestation/code/modules/slimecore/crossbreeding/regenerative/effect.dm

```diff
@@ -0,0 +1,74 @@
+/datum/status_effect/regenerative_extract
+	id = "Slime Regeneration"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 15 SECONDS
+	tick_interval = 0.2 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/regen_extract
+	var/base_healing_amt = 5
+	var/multiplier = 1
+	var/diminishing_multiplier = 0.75
+	var/diminish_time = 45 SECONDS
+	var/nutrition_heal_cap = NUTRITION_LEVEL_FED - 50
+	/// Base traits given to the owner.
+	var/list/given_traits = list(TRAIT_ANALGESIA, TRAIT_NOCRITDAMAGE)
+	/// Extra traits given to the owner, added to the base traits.
+	var/list/extra_traits = list()
+
+/datum/status_effect/regenerative_extract/on_apply()
+	SEND_SIGNAL(owner, COMSIG_SLIME_REGEN_CALC, &multiplier)
+	if(multiplier < 1)
```

Comment:
why does this check multiplier here?

Reply:
it's intended to not have an effect if any cooldowns or active- oh wait, i could just... check status effects